### PR TITLE
fix(DB/Creature): Add a small event for mission 1781

### PR DESCRIPTION
**Pull request by fangshun2004**
## Changes Proposed:
After accepting the quest 1781 , a small event is triggered
Videos from Blizzard servers:
https://github.com/azerothcore/azerothcore-wotlk/assets/49193927/b4236172-db66-4c3d-9f8e-2a96fd45cf93

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [x] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [x] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1. Create an alliance of paladins 
2. .go c 39536
3. .level 12+
4. quest add 1780
5. Complete the quest Accept the new quest 1781
6. Creature 5189 on the podium will heal 6173

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1713019573985621400.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+(@ENTRY * 100, 9, 1, 0, 0, 0, 100, 0, 2000, 2000, 0, 0, 90, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'After 2 seconds - Self: Set stand state to KNEEL'),
+(@ENTRY * 100, 9, 2, 0, 0, 0, 100, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'After 0 seconds - Self: Talk 0 to invoker'),
+(@ENTRY * 100, 9, 3, 0, 0, 0, 100, 0, 2500, 2500, 0, 0, 91, 8, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'After 2.5 seconds - Self: Remove stand state KNEEL'),
+(@ENTRY * 100, 9, 4, 0, 0, 0, 100, 0, 2500, 2500, 0, 0, 66, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'After 2.5 seconds - Self: Set orientation to home position orientation');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 6173 AND `SourceId` = 0;
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 6173;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(6173, 0, 0, 'Thanks to you, $n.', 12, 6, 100, 0, 0, 0, 2284, 0, 'Gazin Tenorm');
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1713019573985621400.sql:34]: missing newline at end of file

File: data/sql/updates/pending_db_world/rev_1713019573985621400.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
+ DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 13) AND (`SourceEntry` IN (8596));
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1713019573985621400.sql:1]: please remove leading space

File: data/sql/updates/pending_db_world/rev_1713019573985621400.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `Comment`) VALUES 
+(13, 1, 8596, 0, 0, 23, 0, 1519, 0, 0, 0, 'Potential target of the spell in area  (1519)'),
+(13, 1, 8596, 0, 0, 31, 0, 3, 6173, 39538, 0, 'Potential target of the spell is creature, entry is Gazin Tenorm (6173) and guid is 39538');
+
+ -- Duthorian Rall smart ai
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1713019573985621400.sql:6]: leading space

File: data/sql/updates/pending_db_world/rev_1713019573985621400.sql [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+SET @ENTRY := 6171;
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` = @ENTRY;
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryOrGuid` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 0, 0, 0, 19, 0, 100, 0, 1781, 0, 0, 0, 86, 8596, 0, 10, 37571, 5489, 0, 10, 39538, 6173, 0, 0, 0, 0, 0, 'On player accepted quest The Tome of Divinity (1781) - Creature Brother Joshua (5489) with guid 37571 (fetching): Cast spell  Heal Visual (8596) at Creature Gazin Tenorm (6173) with guid 39538 (fetching)');
+
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 6171 AND `SourceId` = 0;
+
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1713019573985621400.sql:15]: lots of empty lines - maybe reduce to 1 empty line between blocks?

File: data/sql/updates/pending_db_world/rev_1713019573985621400.sql [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+ -- Gazin Tenorm smart ai
```
Comment (sudlud) [inline data/sql/updates/pending_db_world/rev_1713019573985621400.sql:17]: leading space

**Comment by sudlud** [comment]
just tested this.

```
create human paladin
.level 11
.go c 39536
.quest add 1780
turn in quest "The Tome of Divinity"
accept follow-up quest "The Tome of Divinity"
observe RP
```

looks good :) RP looks like the video provided.

**Review by sudlud — APPROVED** [review]

**Review by pangolp — APPROVED** [review]
Thanks for the contribution

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by pvp100com** [comment]
After my testing, it works very well.

**Comment by pangolp** [comment]
Thanks to all who participated.

**Pull request merged**